.env.test

```diff
@@ -1,7 +1,6 @@
 # define your env variables for the test env here
 KERNEL_CLASS='App\Kernel'
-APP_SECRET='s$cretf0rt3st'
-
-###> sentry/sentry-symfony ###
-SENTRY_DSN=
-###< sentry/sentry-symfony ###
+APP_SECRET='$ecretf0rt3st'
+SYMFONY_DEPRECATIONS_HELPER=999999
```

Comment:
Why? Do we get so much deprecations?

Reply:
Powered by recipes!

Reply:
> Powered by recipes!

I don't get you... was that value added by a recipe? Even if so, does tt make sense for us to keep it like so?
If we don't have internal deprecations, I would set it to `max[self]=0`.
The best values, anyway, is `max[total]=0` (meaning that we don't get any deprecations)

Reply:
done! ;)